pkg/commands/oscommands/os.go

```diff
@@ -267,3 +268,16 @@ func GetLazygitPath() string {
 	}
 	return `"` + filepath.ToSlash(ex) + `"`
 }
+
+func (c *OSCommand) UpdateWindowTitle() error {
+	if runtime.GOOS != "windows" {
+		return nil
+	}
+	path, getWdErr := os.Getwd()
+	if getWdErr != nil {
+		return getWdErr
+	}
+	title := fmt.Sprint(filepath.Base(path), " - Lazygit")
+	args := append([]string{"cmd", "/C", "title"}, strings.Split(title, " ")...)
+	return c.Cmd.NewShell(strings.Join(args, " ")).Run()
```

Comment:
if I check the output in the Command Log panel I see `bash -c "cmd /C title lazygit - Lazygit"`. I'm on osx so it uses bash but I suspect on windows it shows `cmd /C "cmd /C title lazygit - Lazygit`. In effect, the NewShell function adds the `cmd` and `/C` arguments itself. So I'd see how you go removing those two args here

Reply:
Thanks! I've removed the redundant args. Also I noticed that the OSCommand struct has os/platform information so I removed the dependency to the "runtime" package.